nearbridge/package.json

```diff
@@ -25,7 +25,7 @@
         "eslint-plugin-standard": "^4.0.0",
         "eth-gas-reporter": "^0.2.11",
         "ganache-cli": "^6.9.1",
-        "rainbow-bridge-lib": "1.0.0",
+        "rainbow-bridge-lib": "^1.0.0",
```

Comment:
Should we pin a rainbow-bridge-lib version here?

Reply:
I suggest to use the latest version if possible. It helps us to run the latest version in CI and distribute the changes between components quickly.
Do you think it's necessary to pin?

Reply:
Sounds good! I think here it's good to use latest rainbow-bridge-lib and in rainbow-bridge-cli we should pin lib/sol/rs to ensure cli deterministic